Tests/test_file_gimppalette.py

```diff
@@ -29,4 +45,26 @@ def test_get_palette():
     palette, mode = palette_file.getpalette()
 
     # Assert
+    expected_palette = b""
+    for color in (
+        (0, 0, 0),
+        (65, 38, 30),
+        (103, 62, 49),
+        (79, 73, 72),
+        (114, 101, 97),
+        (208, 127, 100),
+        (151, 144, 142),
+        (221, 207, 199),
+    ):
+        expected_palette += bytes(color)
+    assert palette == expected_palette
     assert mode == "RGB"
+
+
+def test_n_colors():
+    # Arrange
+    with open("Tests/images/custom_gimp_palette.gpl", "rb") as fp:
+        palette_file = GimpPaletteFile(fp)
+
+    palette, _ = palette_file.getpalette()
+    assert len(palette) == 24
```

Comment:
```suggestion
```

Now that `n_colors` has been removed, I don't think this is doing anything above and beyond the previous test. Or at least, I think it could be removed from `test_n_colors` to something else.